pkg/controller/che/che_controller.go

```diff
@@ -796,6 +812,27 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 	// Create devfile registry resources unless an external registry is used
 	externalDevfileRegistry := instance.Spec.Server.ExternalDevfileRegistry
 	if !externalDevfileRegistry {
+		if instance.IsAirGapMode() {
```

Comment:
Don't you undate the config map in case it already exists ?
It seems we should, right ?